draft-ietf-quic-recovery.md

```diff
@@ -79,13 +79,40 @@ recovery, and where applicable, attributes the TCP equivalent in RFCs,
 Internet-drafts, academic papers, and/or TCP implementations.
 
 
-## Notational Conventions
+# Conventions and Definitions
 
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
 "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this
 document are to be interpreted as described in BCP 14 {{!RFC2119}} {{!RFC8174}}
 when, and only when, they appear in all capitals, as shown here.
 
+Definitions of terms that are used in this document:
+
+ACK frames:
+
+: ACK frames refer to both ACK and ACK_ECN frames in this
+  document.
+
+ACK-only:
+
+: Any packet containing only an ACK frame.
+
+In-flight:
+
+: Packets are considered in-flight when they have been sent
+  and neither acknowledged or declared lost, and they are not
+  ACK-only.
+
+Retransmittable Frames:
+
+: All frames besides ACK, ACK_ECN, or PADDING are considered
+  retransmittable.
+
+Retransmittable Packets:
+
+: Packets that contain retransmittable frames elicit an ACK from
+  the receiver and are called retransmittable packets.
```

Comment:
Both packets and frames aren't classified as retransmittable or not.  It would be better if this used a different label.

Reply:
@martinthomson : I don't understand your comment. Can you rephrase?

Reply:
The notion that a packet is retransmittable is a misnomer because it creates the impression that you can retransmit the packet.  Similarly, we don't retransmit frames either.  The distinction that is relevant to your uses of these terms is not whether the packet is able to be retransmitted, but whether the packet contains frames other than ACK/ACK_ECN/PADDING.

Reply:
I opened issue #1595 for splitting up the terms, but I'd prefer do that separately.